docs/writing_tests/types_of_tests.md

```diff
@@ -38,3 +50,14 @@ def test_blob_type_tx_pre_fork(
     Reject blocks with blobs before blobs fork
     """
 ```
+
+## Deciding on a test type
+
+### Prefer `state_test` for single transactions
+
+Whenever possible, use `state_test` to examine individual transactions. This method is more straightforward and less prone to external
+influences that can occur during block building.
+
+This provides more targeted testing since it does not invoke the client's block-building machinery. This reduces the risk of
+encountering false positives, particularly in exception scenarios (e.g., see issue
+[#343: "Zero max_fee_per_blob_gas test is ineffective"](https://github.com/ethereum/execution-spec-tests/issues/343)).
```

Comment:
```suggestion
This provides more targeted testing since it does not invoke the client's block-building machinery. This reduces the risk of encountering false positives, particularly in exception scenarios (e.g., see issue
[#343: "Zero max_fee_per_blob_gas test is ineffective"](https://github.com/ethereum/execution-spec-tests/issues/343)).
```